styleguide/components/search/search-docs.mdx

```diff
@@ -12,6 +12,25 @@ import { Story, Preview } from '@storybook/addon-docs/blocks';
   <Story id="components-search--search-with-value" />
 </Preview>
 
+## Using with Rails
+
+If you are using the `citizens_advice_components` gem you call the component from within a template using:
+
+```rb
+= render(CitizensAdviceComponents::Search.new(value: "Current search term"))
+```
+
+Unlike the legacy haml partial the component will only render the input and button, not the surrounding from to give you more control over how the field is used.
```

Comment:
typo: `from` &rarr; `form`

bike shedding but might be better to flip the sentence to emphasis that you'll have to render this in your own `form` as it's a fairly major change from the haml partial.

Reply:
Good shout. The other approach we could take is to allow a `content` block in the form. But based on how you're using this component in public website limiting it to the field seems most flexible.